sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClientBuilder.java

```diff
@@ -81,6 +81,7 @@ public final class TableServiceClientBuilder implements
     HttpTrait<TableServiceClientBuilder>,
     ConfigurationTrait<TableServiceClientBuilder>,
     EndpointTrait<TableServiceClientBuilder> {
+    static private final String suffix = "core.windows.net";
```

Comment:
Do we know if this issue happens exclusively with Storage?

Reply:
@vcolin7 That's a good question. The bug was identified from a customer using Storage but it's very possible that Cosmos could also have this problem. I'll look into it.

Reply:
@vcolin7 The PR was adjust as to only affect Storage endpoints.